cmd/crc/cmd/podman_env.go

```diff
@@ -32,24 +33,16 @@ func runPodmanEnv() error {
 	if err := checkIfMachineMissing(client); err != nil {
 		return err
 	}
-
-	connectionDetails, err := client.ConnectionDetails()
-	if err != nil {
+	p := podman.UsePodman()
```

Comment:
The naming `UsePodman()` is weird (I know it's used elsewhere for OC/...), I was thinking this was testing if something is using podman. Maybe just `podman.Config()`? And one letter variable are quite bad for readability.